src/main/thrift/parquet.thrift

```diff
@@ -194,6 +194,15 @@ enum FieldRepetitionType {
 /**
  * Statistics per row group and per page
  * All fields are optional.
+ *
+ * For BinaryStatistics in Parquet, we want to distinguish between the
+ * statistics derived from comparisons of signed or unsigned bytes.  The min
+ * and max fields are deprecated for BinaryStatistics, instead relying on
+ * specification of {unsigned,signed}_{min,max}. The filter API should allow
+ * clients to specify which statistics and method of comparison should be used
+ * for filtering. To maintain backward format compatibility, when filtering
+ * based on signed statistics the signed_min and signed_max are checked first,
+ * and if they are unset it falls back to using the values in min and max.
```

Comment:
Lets clarify that min and max are always signed if present.

I wonder if any of this belongs in the README too?